partials/html/foot.htm

```diff
@@ -14,7 +14,7 @@
             </p>
         </footer>
 
-        {% snowboard all %}
+        <script src="{{ 'assets/js/app.js'|theme }}"></script>
```

Comment:
```suggestion
        <script src="{{ 'assets/js/theme.js'|theme }}"></script>
```
Minor quibble, but could we use more descriptive names for the JS entrypoints? The generic app.js being used for everything in a project isn't my favourite approach :)

Reply:
Fiiiiiine... done :P